src/index.ts

```diff
@@ -1,37 +1,19 @@
-import { connectDb } from './app';
+import '@extensions/zod/register';
+import { create as createApp } from './app';
 
-connectDb().catch(console.dir);
+async function bootstrap() {
+  // create the app
+  const app = await createApp();
 
-// import { createHttpTerminator } from 'http-terminator';
-// import { create as createApp } from './app';
+  // start server
+  try {
+    await app.listen({ port: 5050, host: '0.0.0.0' });
 
-// function bootstrap() {
-//   // create the app
-//   const { app } = createApp();
+    console.log(`App is running on http://localhost:5050`);
```

Comment:
Here we have access to the `app.log.info()` and we should use it instead of `console.log`